src/wallet/rpcwallet.cpp

```diff
@@ -1756,7 +1773,8 @@ UniValue listsinceblock(const JSONRPCRequest& request)
 
     LOCK2(cs_main, pwallet->cs_wallet);
 
-    const CBlockIndex *pindex = NULL;
+    const CBlockIndex* pindex = NULL;
```

Comment:
These variables might warrant their own comments now, since it's not immediately obvious what they're doing:

pindex - transactions from this block onwards should be included in the result. If the specified block was not in the main chain, pindex is the block where the chain forked
paltindex - used to count back from the specified block to the fork point to collect transactions for the removed array.

You can probably come up with better wording.